src/api/collections-api.tsx

```diff
@@ -0,0 +1,14 @@
+import { getConfig, CollectionsApi, ListCollectionsResponse } from "@imtbl/core-sdk";
+
+// Get a list of collections
+// Inspired from Immutable Core SDK documentation
+const GetYourCollections = async (): Promise<ListCollectionsResponse> => {
+  const config = getConfig("ropsten");
+  const collectionsApi = new CollectionsApi(config.api);
+
+  const response = await collectionsApi.listCollections()
+
+  return response.data;
+};
+
+export default GetYourCollections
```

Comment:
please remove default export and add export to your function as you name your file as collections-api, it means that you can have more than one method here.